.github/workflows/docs.yml

```diff
@@ -0,0 +1,58 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "Build documentation"
+on:
+  workflow_dispatch:
+jobs:
+  build-documentation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Set branch environment variable
+        run: |
+          currentBranch=$(git branch --show-current)
+
+          echo "flink_branch=${currentBranch}" >> ${GITHUB_ENV}
+
+          if [ "${currentBranch} = "master" ]; then
+            echo "flink_alias=release-1.16" >> ${GITHUB_ENV}
```

Comment:
we don't have it only because you guys didn't update the buildbot setup when creating the 1.15 release branch :)

Reply:
IIRC we always did that when creating the release branch.
although admittedly I can't find it in the release guide.

Reply:
Ah there it is; `Create a release branch#Major Release`:

> After pushing the new major release branch, as the last step you should also update the documentation build bot to also build the documentation for the new release branch. Check [Managing Documentation](https://cwiki.apache.org/confluence/display/FLINK/Managing+Documentation) on details on how to do that. You may also want to manually trigger a build to make the changes visible as soon as possible.

Reply:
Ah I also didn't find it, because I read "Major Release" and assumed it was applicable for a 1.x to 2.x release. Perhaps we should create some sort of checklist out of all these steps you can follow if you've completed all the steps

Reply:
You mean like item 13 in the `Prepare for the release` section? :)

> (major only) The documentation for the new major release is visible under https://nightlies.apache.org/flink/flink-docs-release-$SHORT_RELEASE_VERSION (after at least one doc build finishes).

Note that in terms of Flink, major refers to 1.X.0, while miner are 1.Y.X. We don't have a term for 2.x. It's stupid, I know.

Reply:
> You mean like item 13 in the checklist of the `Prepare for the release` section? :)

Yes but then using Jira's task options, so you could also create an overview of the open tasks. But that could be overkill given that anyone who works on a release should read it properly and if you've done it once, you'll probably remember it.